doc/_themes/sphinx13/layout.html

```diff
@@ -27,28 +27,25 @@
     </style>
     <script>
       // intelligent scrolling of the sidebar content
-      $(window).scroll(function() {
-        var sb = $('.sphinxsidebarwrapper');
-        var win = $(window);
-        var sbh = sb.height();
-        var offset = $('.sphinxsidebar').position()['top'];
-        var wintop = win.scrollTop();
-        var winbot = wintop + win.innerHeight();
+      jQuery(window).scroll(function() {
+        var sb = document.getElementsByClassName('sphinxsidebarwrapper')[0]
+        var sbh = sb.offsetHeight
+        var offset = jQuery('.sphinxsidebar').position()['top'];
```

Comment:
This still depends on jQuery. Is that intentional?

Reply:
Sorry! Just pushed a commit to fix this.

A